cfpb/cfgov-refresh
# Migrate pull quote to cfv4

## Changes

- Updates `m-quote` and `pull-quote` to Capital Framework v4 `m-pull-quote`.
- Adds "is large" option to Wagtail to include large pull quote style from CF.

## Testing

1. Create a test page in Wagtail.
2. Add a full-width text component.
3. Add a quote to the full-width text.
4. Preview changes to see pull quote.

## Screenshots

Before:
![screen shot 2017-06-23 at 6 22 25 pm](https://user-images.githubusercontent.com/704760/27502733-5ba176d4-5844-11e7-82b0-58ed0f188c26.png)

After:
![screen shot 2017-06-23 at 6 15 53 pm](https://user-images.githubusercontent.com/704760/27502740-60a10b9a-5844-11e7-8e91-e01c1ddfdff6.png)

Wagtail:
![screen shot 2017-06-23 at 6 42 52 pm](https://user-images.githubusercontent.com/704760/27502743-66a6771e-5844-11e7-9b54-e8ed62f5f98b.png)

## Todos

- This probably needs tests for the wagtail changes, but I need guidance on how to add them. Anyone?
